include/TrustWalletCore/TWAnySigner.h

```diff
@@ -23,6 +23,12 @@ extern TWString *_Nonnull TWAnySignerSignJSON(TWString *_Nonnull json, TWData *_
 
 extern bool TWAnySignerSupportsJSON(enum TWCoinType coin);
 
+/// Encodes a broadcastable transaction to raw bytes
```

Comment:
As both input&output are Data type, their type does not tell much, comment is important.  What is 'raw bytes'?  I think a bit more specific description is needed here.